locales/en/settings.ftl

```diff
@@ -0,0 +1,45 @@
+# Settings page
```

Comment:
I haven't been strict about FTL files in this repository, but since this is a new file, I think it's worth cleaning up the formatting.

This is how the file would be formatted in mozilla-central:
* License header
* Spaces around message/term/variable references
* Mandatory comments for variables

Could you update this file accordingly? I'm attaching the updated FTL (renamed to .txt to be able to attach it here).

[settings.txt](https://github.com/mozilla/blurts-server/files/10529118/settings.txt)

I'd  like to enforce this style going forward for new strings.

Reply:
Thank you @flodolo for putting together example FTL — updated and noted for any new additions.